test/cypress/integration/all-service-transactions/all-service-transactions.cy.test.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const userStubs = require('../../stubs/user-stubs')
+const gatewayAccountStubs = require('../../stubs/gateway-account-stubs')
+const transactionStubs = require('../../stubs/transaction-stubs')
+const stripeAccountSetupStubs = require('../../stubs/stripe-account-setup-stub')
+
+describe('Transactions list pagination', () => {
```

Comment:
Can this be renamed to better describe what the test is for?

Reply:
Good spot - done